forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt

```diff
@@ -65,6 +65,14 @@ class ForagePINEditText @JvmOverloads constructor(
     private val vgsVaultWrapper: VGSVaultWrapper
     private val forageVaultWrapper: ForageVaultWrapper
 
+    override fun showKeyboard() {
+        if (vault.getVaultType() == VaultType.VGS_VAULT_TYPE)
+            vault.getVGSEditText().showKeyboard()
+        if (vault.getVaultType() == VaultType.BT_VAULT_TYPE) {
+            TODO("currently blocked by BT's lack of this functionality")
+        }
```

Comment:
[Throwing here](https://arc.net/l/quote/dlspxekb) may be danagerous ; can we pass/do nothing instead